shapeshift/web
# fix: show correct staking balance by account id

## Description



previously we were aggregating all staked balance by asset id, this adds a selector and filters by both asset and account id's

## Notice




## Pull Request Type


## Issue (if applicable)



related to but does not close #2614 

## Risk



* we show incorrect staked amounts above the chart
* we show incorrect cosmossdk account balances on the accounts page

## Testing



* turn multi account flag on
* add a cosmos account
* navigate to new cosmos account (can have balance or not)
* new cosmos account should not show any staked balance from account 0. if new account has a staked balance it should show that amount correctly.

### Engineering



* see common testing steps

### Operations



* see common testing steps

## Screenshots (if applicable)

before (account with a single tx and no staked balance)

<img width="746" alt="image" src="https://user-images.githubusercontent.com/88504456/191335990-b5514890-d309-447b-8dab-ae145dadcb98.png">

after with same account

<img width="760" alt="image" src="https://user-images.githubusercontent.com/88504456/191335670-6da0fd30-235d-4437-a126-78e752c654fc.png">
